x-pack/plugins/uptime/public/components/functional/__tests__/filter_bar.test.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { shallowWithIntl } from 'test_utils/enzyme_helpers';
+import { FilterBar as FBType } from '../../../../common/graphql/types';
+import { FilterBar } from '../filter_bar';
+
+describe('FilterBar component', () => {
+  const filterBar: FBType = {
+    status: ['up', 'down'],
+    port: [80, 443, 9200, 12349],
+    id: [
+      'http@http://localhost:12349/',
+      'http@http://www.example.com',
+      'http@http://www.google.com/',
+      'http@https://news.google.com/',
+      'http@https://www.elastic.co',
+      'http@https://www.github.com/',
+      'http@https://www.google.com/',
+      'http@https://www.wikipedia.org/',
+      'tcp-tcp@localhost:9200',
+    ],
+    type: ['http', 'tcp'],
```

Comment:
Should we add an ICMP example here?

Reply:
👍 I will add a new API test for the same reason.

Reply:
Please check https://github.com/elastic/kibana/pull/32083/commits/63b23374240da79f599b8918ed74c34c2f1e9b1b.